### PR TITLE
Implemented CommonCoreStandards in Item Info tab.

### PR DESCRIPTION
**Pull request by bgsavage**
Added commonCoreStandard controller/service/model/repo.
Calling commonCoreStandard api from ui and display the results in the item-info tab

### CODE REVIEW

File: webapp/src/main/webapp/src/app/assessments/items/item-info/item-info.component.ts [hunk 1]
```diff
@@ -13,17 +12,23 @@ export class ItemInfoComponent implements OnInit {
 
   interpretiveGuide: string;
   targetDescription: string;
+  commonCoreStandards: any[];
 
-  constructor(private userService: UserService, private cachingDataService: CachingDataService) { }
+  constructor(private service: ItemInfoService) { }
```
Comment (bgsavage) [inline webapp/src/main/webapp/src/app/assessments/items/item-info/item-info.component.ts]: unrelated: moved logic out of the component and into an `ItemInfoService`

File: webapp/src/main/webapp/src/app/assessments/results/assessment-results.component.html [hunk 2]
```diff
@@ -179,7 +179,7 @@ <h3 class="gray-darkest mb-sm">
           </p-column>
           <p-column field="commonCoreStandardIds" header="{{'labels.groups.results.assessment.items.cols.standard' | translate}}" sortable="true">
             <ng-template let-standards="rowData.commonCoreStandardIds" pTemplate="body">
-              <span *ngFor="let standard of standards" class="label border-only blue">{{standard}}</span>
+              <span *ngFor="let standard of standards" class="label border-only blue mr-xs">{{standard}}</span>
```
Comment (bgsavage) [inline webapp/src/main/webapp/src/app/assessments/results/assessment-results.component.html]: unrelated, added some spacing to the common core standards in the items by points earned table as they were looking a bit squished.

File: webapp/src/main/java/org/opentestsystem/rdw/reporting/standard/CommonCoreStandard.java [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+            return this;
+        }
+
+        public Builder description(String description) {
+            this.description = description;
+            return this;
+        }
+
+        public CommonCoreStandard build() {
+            CommonCoreStandard standard = new CommonCoreStandard();
```
Comment (malaffoon) [inline webapp/src/main/java/org/opentestsystem/rdw/reporting/standard/CommonCoreStandard.java]: nit: final

File: webapp/src/main/java/org/opentestsystem/rdw/reporting/standard/CommonCoreStandardController.java [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
+public class CommonCoreStandardController {
+    private CommonCoreStandardService service;
```
Comment (malaffoon) [inline webapp/src/main/java/org/opentestsystem/rdw/reporting/standard/CommonCoreStandardController.java]: nit: final (also all parameters) ... okay, i won't mention this again.

**Comment by malaffoon** [comment]
Could use some unit tests of course ...

**Comment by bgsavage** [comment]
I don't find javadocs particularly useful here as it's just echoing the property names, but I added them....

Also made the classes package private and did all the finals.

**Comment by bgsavage** [comment]
@malaffoon  It seems the preferred approach in this project for testing is IT, so I'll be adding IT in the next pr

**Pull request merged**